### PR TITLE
Econ 2.0

### PR DESCRIPTION
**Pull request by gawquon**
- [ ] Add demand factor
- [ ] Rebalance weights
- [x] ~~Verify used defines~~

---
### Demand Pop Component
- [x] Load goods data
- [x] Load popneeds
- [x] Load buy packages
- [x] Load poptype data (consumption multiplier & base dependent ratio)
- [x] Load WORKING_ADULT_RATIO_BASE
- [x] Load DEPENDENT_CONSUMPTION_RATIO
- [x] Add formula to EconManager
### Demand Building Component
- [x] ~~Load employment from PMs~~ (Already did, thanks past me.)
- [x] Load input/output goods from PMs
- [ ] Use average weight of PMs from EconManager config file. (Country by country basis)
- [ ] Predict ownership PMs based on country laws
- [ ] Track available workers and Subsistence space.

### CODE REVIEW

File: EU4ToVic3/Source/V3World/ClayManager/State/State.cpp [hunk 1]
```diff
@@ -14,9 +14,6 @@ void V3::State::loadState(std::istream& theStream)
 	parseStream(theStream);
 	clearRegisteredKeywords();
 	updateProvinces();
-
```
Comment (gawquon) [inline EU4ToVic3/Source/V3World/ClayManager/State/State.cpp:17]: Pretty sure there is no reason for these to be here. The SubStates are always empty at the time of calling loadState, and these functions work over the SubStates. They're handled by the ClayManager.

File: EU4ToVic3/Source/V3World/EconomyManager/Building/ProductionMethods/ProductionMethod.cpp [hunk 2]
```diff
@@ -12,56 +12,58 @@ void V3::ProductionMethod::loadProductionMethod(std::istream& theStream)
 
 void V3::ProductionMethod::registerKeys()
 {
+	scalingParser.registerKeyword("country_bureaucracy_add", [this](std::istream& theStream) {
+		bureaucracy = commonItems::getInt(theStream);
+	});
+	scalingParser.registerRegex("goods_input\\w+_add", [this](const std::string& goodsType, std::istream& theStream) {
+		inputs.emplace(getType(goodsType), commonItems::getInt(theStream));
```
Comment (Zemurin) [inline EU4ToVic3/Source/V3World/EconomyManager/Building/ProductionMethods/ProductionMethod.cpp:19]: Let's hope they never introduce a method that produces or uses *half* of a good. Maybe long term doubles would be better but it's your call.

**Review by Zemurin — APPROVED** [review]
I keep reading the market and market tests and it's like reading a fairy tale. Next release will be named A PopDemand Fairy Tale.

File: EU4ToVic3/Source/V3World/EconomyManager/Building/ProductionMethods/ProductionMethod.cpp (hunk 2 above)
Comment (gawquon) [reply EU4ToVic3/Source/V3World/EconomyManager/Building/ProductionMethods/ProductionMethod.cpp:19]: Subsistence Buildings already have fractional PMs, good call.

**Pull request merged**